azure-devops/azext_devops/dev/repos/repository.py

```diff
@@ -93,10 +94,49 @@ def list_repos(organization=None, project=None, detect=None):
         raise CLIError(ex)
 
 
-def show_repo(repo, organization=None, project=None, detect=None, open=False):  # pylint: disable=redefined-builtin
+def update_repo(repository, default_branch=None, name=None, organization=None, project=None, detect=None):
+    """Update the Git repository.
+    :param repository: Name or ID of the repository.
+    :type repository: str
+    :param organization: Azure Devops organization URL. Example: https://dev.azure.com/MyOrganizationName/
+    :type organization: str
+    :param project: Name or ID of the team project.
+    :type project: str
+    :param name: New name for the repository.
```

Comment:
This is little bit confusing.
Basically just by looking at name it is hard to tell which params are for update and which are for identification

Reply:
@gauravsaralMs  Similar usage in teams commands. That is where I picked up from. 
@geverghe  What should be the param for rename in general: --name or --new-name? 

![image](https://user-images.githubusercontent.com/11888714/51836339-2cde3400-2326-11e9-9d80-5c104e5693cc.png)

Reply:
Filed issue #388. Merging the PR.